src/unlimited.cpp

```diff
@@ -553,40 +510,15 @@ void UnlimitedSetup(void)
     if (Params().NetworkIDString() == "main")
     {
         CCheckpointData &checkpoints = ModifiableParams().ModifiableCheckpoints();
-        if (nMiningSvForkTime == 0)
-        {
-            // Nov 15th 2018 activate LTOR, DSV op_code
-            checkpoints.mapCheckpoints[556767] =
-                uint256S("0000000000000000004626ff6e3b936941d341c5932ece4357eeccac44e6d56c");
-            // * UNIX timestamp of last checkpoint block
-            checkpoints.nTimeLastCheckpoint = 1542304936;
-            // * total number of transactions between genesis and last checkpoint
-            checkpoints.nTransactionsLastCheckpoint = 265567564;
-            // * estimated number of transactions per day after checkpoint (~3.5 TPS)
-            checkpoints.fTransactionsPerDay = 280000.0;
-        }
-        else if (nMiningSvForkTime != 0)
-        {
-            // Nov 15th 2018 SV fork, 128MB blocks, re-enable bitcoin 0.1.0 op_codes
-            checkpoints.mapCheckpoints[556767] =
-                uint256S("000000000000000001d956714215d96ffc00e0afda4cd0a96c96f8d802b1662b");
-            // * UNIX timestamp of last checkpoint block
-            checkpoints.nTimeLastCheckpoint = 1542305817;
-            // * total number of transactions between genesis and last checkpoint
-            checkpoints.nTransactionsLastCheckpoint = 265615408;
-            // * estimated number of transactions per day after checkpoint (~3.5 TPS)
-            checkpoints.fTransactionsPerDay = 280000.0;
-        }
-        else
-        {
-            // unknown scenario, dont update these values or add a new checkpoint
-            // * UNIX timestamp of last checkpoint block
-            checkpoints.nTimeLastCheckpoint = 1526410186;
-            // * total number of transactions between genesis and last checkpoint
-            checkpoints.nTransactionsLastCheckpoint = 249416375;
-            // * estimated number of transactions per day after checkpoint (~3.5 TPS)
-            checkpoints.fTransactionsPerDay = 280000.0;
-        }
+        // Nov 15th 2018 activate LTOR, DSV op_code
+        checkpoints.mapCheckpoints[556767] =
+            uint256S("0000000000000000004626ff6e3b936941d341c5932ece4357eeccac44e6d56c");
+        // * UNIX timestamp of last checkpoint block
+        checkpoints.nTimeLastCheckpoint = 1542304936;
+        // * total number of transactions between genesis and last checkpoint
+        checkpoints.nTransactionsLastCheckpoint = 265567564;
+        // * estimated number of transactions per day after checkpoint (~3.5 TPS)
+        checkpoints.fTransactionsPerDay = 280000.0;
     }
```

Comment:
I think we should move this chunck of code back in `src/chainparams.cpp`, basically reverting @ptschip's c891c1478f840b8333b14b833a3d1ef5d0f5abfc

Reply:
I will make your suggested change if @ptschip doesn't have a problem with it.

Reply:
we have to make checkpoints modifiable because we need to change their value after initialization. this need stem from the fact that we supported 2 different chains in the same code base.

This is not the case anymore. IMHO having the code back in `chainparams.cpp` let you have all checkpoints data in the same place (like it was before) rather than having them scattered in 2 different files.

Not forcing you to change it, just interested your opinion on it.

Reply:
I think they should be moved back as well. I just did not want to undo his PR without the authors input.

Reply:
> I just did not want to undo the PR without the authors input
fair. 

> However, with your explaination on why it was changed i will revert it back now.
thanks :smile:

Reply:
Just one last thing, we should also remove line 97: 

```c++
CCheckpointData &ModifiableCheckpoints() { return checkpointData; }
```

from `src/chainparams.h`